<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionUpdate</class>
 <widget class="QWidget" name="VersionUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>711</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="insertButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>新增</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上一级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>装备名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
